docs/source/troubleshooting.rst

```diff
@@ -78,3 +78,16 @@ Things to check:
     date > date.txt
     gsutil cp date.txt ${YOUR_RESULTS_BUCKET}
     
+
+I get a message about a project not existing
+--------------------------------------------
+
+Things to check:
+
+* Make sure you are using the GCP project ID.  Every GCP project has a name, an ID and a number.  The name and ID consist of lower-case letters and dashes and possibly numbers.  The project number is simply an integer.  See all three by going to your dashboard at https://console.cloud.google.com/home/dashboard
+
+* Verify that you have the correct project set with:
+
+.. code-block:: bash
+
+   gcloud config get-value project
```

Comment:
This is wrong. Default gcloud project does not need to be set for elastic-blast to work and many users may have it unset. A user would need to check that project id in GCP console is the same as the one give in elastic-blast config.

Reply:
OK